revpi_core.c

```diff
@@ -289,10 +289,14 @@ static int init_connect_gpios(struct platform_device *pdev)
 		return PTR_ERR(piCore_g.gpio_x2do);
 	}
 
-	piCore_g.gpio_wdtrigger = devm_gpiod_get_index(&pdev->dev, "connect", 2, GPIOD_OUT_LOW);
-	if (IS_ERR(piCore_g.gpio_wdtrigger)) {
-		dev_err(&pdev->dev, "cannot acquire gpio watchdog trigger\n");
-		return PTR_ERR(piCore_g.gpio_wdtrigger);
+	/* WD trigger is only needed on Connect / Connect SE */
```

Comment:
The code comment isn't really helpful because the subsequent if-clause already implies that.
Perhaps change to "is not needed on Connect 4"?